docs/UserGuide.md

```diff
@@ -147,6 +153,11 @@ Sample Excel file:
 How to identify import button:
 ![Import_Button](images/Import_Button.png) *Figure 4: Screenshot of import button.*
 
+⚠️`Warning:`
+* The excel file must contain only one sheet named "Sheet1".
+* The column of the table must be capital.
+* The data in the table needs to be ensured no space behind
+
```

Comment:
I think it will be good to include a screenshot of a correct excel file and also that the excel file must have extension of xlsx (or anything else with works)

Reply:
Good suggestion! will add it